diff --git a/cmake/gazebo-config.cmake.in b/cmake/gazebo-config.cmake.in
index b39b469..b5cc84c 100644
--- a/cmake/gazebo-config.cmake.in
+++ b/cmake/gazebo-config.cmake.in
@@ -104,7 +104,7 @@ execute_process(COMMAND pkg-config --variable=prefix OGRE
 
 # Add the OGRE cmake path to CMAKE_MODULE_PATH
 set(CMAKE_MODULE_PATH
-  "${OGRE_INSTALL_PREFIX}/share/OGRE/cmake/modules;${OGRE_INSTALL_PREFIX}/lib/OGRE/cmake;${OGRE_INSTALL_PREFIX}/CMake;${CMAKE_MODULE_PATH}")
+  "${OGRE_INSTALL_PREFIX}/share/OGRE/cmake/modules;${OGRE_INSTALL_PREFIX}/lib64/OGRE/cmake;${OGRE_INSTALL_PREFIX}/CMake;${CMAKE_MODULE_PATH}")
 
 foreach(dep @PKG_DEPENDS@)
   if(NOT ${dep}_FOUND)
