--- pdf-presenter-console-4.0.5.orig/src/classes/window/presenter.vala
+++ pdf-presenter-console-4.0.5/src/classes/window/presenter.vala
@@ -163,7 +163,7 @@ namespace pdfpc.Window {
                 metadata,
                 current_allocated_width,
                 (int) Math.floor(Options.current_height * bottom_position / (double) 100),
-                Metadata.Area.NOTES,
+                Metadata.Area.CONTENT,
                 Options.black_on_end,
                 true,
                 this.presentation_controller,
@@ -183,7 +183,7 @@ namespace pdfpc.Window {
                 metadata,
                 next_allocated_width,
                 (int) Math.floor(Options.next_height * bottom_position / (double)100 ),
-                Metadata.Area.CONTENT,
+                Metadata.Area.NOTES,
                 true,
                 false,
                 this.presentation_controller,
@@ -423,9 +423,9 @@ namespace pdfpc.Window {
             try {
                 this.current_view.display(current_slide_number);
                 this.next_view.display(this.metadata.user_slide_to_real_slide(
-                    current_user_slide_number + 1));
+                    current_user_slide_number));
                 if (this.presentation_controller.skip_next()) {
-                    this.strict_next_view.display(current_slide_number + 1, true);
+                    this.strict_next_view.display(current_slide_number, true);
                 } else {
                     this.strict_next_view.fade_to_black();
                 }
@@ -461,7 +461,7 @@ namespace pdfpc.Window {
         public void goto_page(int page_number) {
             try {
                 this.current_view.display(page_number);
-                this.next_view.display(page_number + 1);
+                this.next_view.display(page_number);
             } catch( Renderer.RenderError e ) {
                 error("The pdf page %d could not be rendered: %s", page_number, e.message);
             }
